keps/sig-api-machinery/555-server-side-apply/README.md

```diff
@@ -349,8 +524,11 @@ This can be promoted to beta when it is a drop-in replacement for the existing
 kubectl apply, and has no regressions (which aren't bug fixes). This KEP will be
 updated when we know the concrete things changing for beta.
 
-This will be promoted to GA once it's gone a sufficient amount of time as beta
-with no changes. A KEP update will precede this.
+A GA version of this is targeted for 1.21.
+
+- E2E tests are created and graduate to conformance
+- [Apply for client-go's typed client](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2144-clientgo-apply) is implemented and has an in-tree controller using it
+- Outstanding bugs around status wiping and scale subresource are fixed
```

Comment:
We had verbally agreed to require a client-go client and using that client in at least one kube-controller-manager controller.  I believe both are on track from @jpbetz .  Can we add those here?

Reply:
Line 530 should cover the client-go client and the controller. Do you want it to be stated more explicitly?